harbor-helm/values.yaml

```diff
@@ -691,3 +691,34 @@ redis:
     password: ""
   ## Additional deployment annotations
   podAnnotations: {}
+
+# Tests that can be executed by using helm test
+tests:
+  api:
+    # Enable/disable API tests
+    enabled: true
+    # Stops test execution if any test fails
+    abortOnFailure: false
+    # Select test cases to run by tag
+    whitelist: []
```

Comment:
BTW, if you try to push this upstream, you'll have to rename `whitelist` and `blacklist` to something like `allowlist` and `denylist` or `includelist` and `excludelist`

Reply:
good point, I have changed it to `include` and `exclude`which are the same parameters used by robotframework.